cockroachcloud/pci-dss.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Payment Card Industry Data Security Standard (PCI DSS) Compliance in CockroachDB Dedicated
+summary: Learn about compliance with the Payment Card Industry Data Security Standard (PCI DSS) for CockroachDB Dedicated clusters.
+toc: true
+docs_area: manage.security
+---
+
+The Payment Card Industry Data Security Standard (PCI DSS) is a minimum set of requirements for the safe handling of sensitive data associated with credit and debit cards. In the PCI DSS standard, this data is referred to as "cardholder data". When implemented correctly, PCI DSS helps to protect cardholder data from fraud, exfiltration, and theft. PCI DSS is mandated by credit card issuers but administered by the [Payment Card Industry Security Standards Council](https://www.pcisecuritystandards.org/).
+
+Many organizations who do not store cardholder data still rely upon compliance with PCI DSS to help protect other sensitive or confidential data or metadata.
+
+Responsibility for compliance with PCI DSS is shared among multiple parties, including card issuers, banks, software-as-a-service (SaaS) providers, and retail merchants. Compliance measures are implemented as a series of business practices, security controls, and technological solutions. An organization's compliance with PCI DSS is certified by a PCI Qualified Security Assessor (QSA).
+
+CockroachDB Dedicated has been certified by a QSA as a PCI DSS [Level 1 Service Provider](https://www.pcidssguide.com/what-are-pci-service-provider-compliance-levels/). This certification extends the existing [SOC 2 Type 2 certification](https://www.cockroachlabs.com/blog/soc-2-compliance-2/) of {{ site.data.products.dedicated }}, which provides a baseline level of security controls to safeguard customer data.
+
+This page provides information about {{ site.data.products.dedicated }}'s compliance with PCI DSS, describes some of the ways that {{ site.data.products.db }} implements and enforces compliance, and illustrates some of the types of changes you may need to implement outside of your {{ site.data.products.dedicated }} clusters.
+
+## Overview of PCI DSS
+
+When a system complies with PCI DSS, the system meets the goals of the standard by implementing a series of requirements, as assessed by an independent PCI Qualified Security Assessor (QSA). The following table, which is published in Payment Card Industry Security Standards Council's [PCI DSS Quick Reference Guide, version 3.2.1](https://listings.pcisecuritystandards.org/documents/PCI_DSS-QRG-v3_2_1.pdf), summarizes the goals and requirements of PCI DSS.
+
+<table>
+<tgroup cols="2">
+<thead>
+<tr>
+  <th>Goal</th>
+  <th>PCI DSS Requirement</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+  <td>Build and Maintain a Secure Network and Systems</td>
```

Comment:
For the items in this column (Goal):

- I'm not sure they need to be in title case. Maybe they can just be sentence case?
- Append each line with a period to match the enumerated list in the next column? Our style may be vague on this.

I realize you're replicating the table in the PCI DSS reference, but it should probably match our style here.